website/src/components/LastUpdated/index.tsx

```diff
@@ -0,0 +1,28 @@
+import fs from "fs";
+import path from "path";
+
+export default function LastUpdated({ dirname }: { dirname: string }) {
+  // timestamps.json was generated during build
+  const timestampsFile = path.resolve("timestamps.json");
```

Comment:
Any risk of the `timestamps.json` file not being there?

Reply:
Yeah, if it's missing, the Vercel build will fail CI and we can go investigate. I hit that last night so it seemed best to just "let it fail" hard

Reply:
Gotcha.  Makes sense.